src/Features/LanguageServer/Protocol/Features/Options/QuickInfoOptionsStorage.cs

```diff
@@ -15,8 +15,10 @@ public static QuickInfoOptions GetQuickInfoOptions(this IGlobalOptionService glo
               IncludeNavigationHintsInQuickInfo = globalOptions.GetOption(IncludeNavigationHintsInQuickInfo),
           };
 
+        private static readonly OptionGroup s_quickInfoGroup = new(name: "Quick Info", description: "");
```

Comment:
>Quick Info [](http://example.com/codeflow?start=74&length=10)

Do we want spaces in the option names?

I think it would be better not to.

Also, it would probably be better if we used the same naming convention for group names as we do for option names. That is `quick_info`. Then we could translate both group names and option names to camelCase using the same alg.